back/engines/commercial/idea_custom_fields/app/controllers/idea_custom_fields/web_api/v1/admin/idea_custom_fields_controller.rb

```diff
@@ -31,6 +31,9 @@ class WebApi::V1::Admin::IdeaCustomFieldsController < ApplicationController
     def index
       authorize CustomField.new(resource: @custom_form), :index?, policy_class: IdeaCustomFieldPolicy
       fields = IdeaCustomFieldsService.new(@custom_form).all_fields
+
+      fields = fields.support_free_text_value if params[:support_free_text_value].present?
```

Comment:
When there is no custom form, we fall back to an array (not a scope) of default custom fields. I suspect that you will have a "no method support_free_text_value for type Array" error in that case. If not, I think we still need a spec to verify that this won't happen.

Reply:
Thanks, I didn't get that, indeed.